### PR TITLE
chore(TDI-49532): use custom builder

### PR DESCRIPTION
**Pull request by acatoire**
https://jira.talendforge.org/browse/TDI-49532
Use custom builder as rootless on scan and build jobs

### CODE REVIEW

**Review by lxia-talend — APPROVED** [review]
LGTM

File: ci/Jenkinsfile-scan [hunk 1]
```diff
@@ -59,58 +43,185 @@ spec:
         skipStagesAfterUnstable()
     }
 
-    triggers {
```
Comment (lxia-talend) [inline ci/Jenkinsfile-scan:62]: no need trigger now ?

**Review by lxia-talend — APPROVED** [review]
LGTM

File: ci/Jenkinsfile-scan (hunk 1 above)
Comment (acatoire) [reply ci/Jenkinsfile-scan:62]: yes triggers are inside the flux config

**Pull request merged**